aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -436,94 +427,96 @@ out_int8 = (act_fp32 * w_fp32 + [bias_fp32]) / out_scale + out_zero_point
              = (act_int8 * w_int8 + [bias_fp32/(act_scale * w_scale)]) / (out_scale / (act_scale * w_scale))
              = requantize((act_int8 * w_int8 + [bias_fp32/(act_scale * w_scale)]), out_scale / (act_scale * w_scale))
 */
-template <int kSpatialDim, bool kReluFused>
-Tensor raw_cudnn_convolution_forward(
-    const Tensor& act,
-    const Tensor& weight,
-    c10::optional<Tensor> bias,
-    IntArrayRef padding,
-    IntArrayRef stride,
-    IntArrayRef dilation,
-    int64_t groups,
-    bool benchmark,
-    bool deterministic,
-    bool allow_tf32,
-    float bias_multiplier,
-    float requantize_multiplier,
+template <int kSpatialDim>
+template <bool kReluFused>
+at::Tensor PackedConvWeightCudnn<kSpatialDim>::apply_impl(
+    const at::Tensor& act,
     double output_scale,
     int64_t output_zero_point) {
-  // TODO: add dimension validations for input/weight/bias
   const int N = act.size(0);
   const int D = kSpatialDim == 3 ? act.size(2) : 1;
   const int H = act.size(kSpatialDim);
   const int W = act.size(kSpatialDim + 1);
-  const int M = weight.size(0); // output channels
-  std::vector<int64_t> kernel_size = {weight.size(2), weight.size(3)};
-  at::SmallVector<int64_t, kSpatialDim + 2> output_shape{MakeConvOutputShape<kSpatialDim>(N, M, {H, W},
-  kernel_size, stride, padding, dilation)};
-  Tensor quantized_output = at::_empty_affine_quantized(
+  const int M = orig_weight.size(0); // output channels
```

Comment:
nit: is this the saved weight? should we give it a special name to make it more explicit? like `weight_`?

Reply:
same for `bias` I think

Reply:
Yeah orig_weight is the saved weight. qnnpack also used the same convention with no underscore. 
I can append the underscore and then have another PR that does the same for qnnpack for uniformity. 

I can do the same for `bias`. I foudn that the naming convention isn't consistent, e.g., in https://github.com/pytorch/pytorch/blob/gh/dzdang/42/head/aten/src/ATen/native/quantized/cpu/fbgemm_utils.h
you can see both `bias` (for `PackedConvWeight` class) and `bias_` (for `PackedLinearWeight` class) used for member variables

Reply:
yeah we should fix them I think